1962 refactor/add icons to upload modal

Ryoken opened the pull request:
#### Any background context you want to provide?

#### What's this PR do?
Re-arranges the data upload modal to make a bit more sense and include icons.

#### How should this be manually tested?

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/1962

#### Screenshots (if appropriate)
<img width="617" alt="Screen Shot 2021-08-09 at 1 07 31 PM" src="https://user-images.githubusercontent.com/564231/128761199-1ce8fee6-bdde-42ea-b127-d2cfb6609e57.png">
<img width="553" alt="Screen Shot 2021-08-09 at 1 07 40 PM" src="https://user-images.githubusercontent.com/564231/128761203-eb99d09e-5318-41cf-9de1-ac7d95f1776a.png">
<img width="620" alt="Screen Shot 2021-08-09 at 1 08 38 PM" src="https://user-images.githubusercontent.com/564231/128761197-877cf65a-71a1-4e30-8d87-d61909546f4c.png">
<img width="621" alt="Screen Shot 2021-08-09 at 1 08 43 PM" src="https://user-images.githubusercontent.com/564231/128761204-7f9aaad5-cbe2-4339-8012-ff5088afae27.png">

Ryoken commented:
Updated based on comments on the ticket.

macintoshpie commented on seed/static/seed/partials/data_upload_modal.html:
```diff
@@ -30,50 +30,113 @@ <h4 class="modal-title" ng-switch-when="16" translate>PM_METER_IMPORT_RESULTS</h
         </div>
     </div>
     <!-- Step 2: Choose upload type -->
```
Maybe we should update this comment now that this handles step `2` and step `13`

macintoshpie reviewed (commented):
Tested all flows I could think of (and error cases) and everything looks great! One suggestion though is tweaking how we display things while a file is being uploaded. E.g. this is what a user sees right after selecting a file for upload. Maybe we should hide the entire tab section instead of just the buttons?

<img width="580" alt="Screen Shot 2021-08-13 at 3 31 53 PM" src="https://user-images.githubusercontent.com/18518728/129410791-c21cc569-4b57-4876-a40d-e8a34896751e.png">

macintoshpie commented:
Oh yeah, also could you update the screenshots now that we've moved the cycles? Sorry I forgot to while testing 😬

Ryoken commented:
Moved cycle:
<img width="621" alt="Screen Shot 2021-08-16 at 12 56 38 PM" src="https://user-images.githubusercontent.com/564231/129620066-8a6ebad7-0611-4cda-aed4-5e58a73916c5.png">

Cleaned up during upload:
<img width="627" alt="Screen Shot 2021-08-16 at 1 36 18 PM" src="https://user-images.githubusercontent.com/564231/129620043-c3093c48-88ed-4bb2-a5d9-bd936c87a46e.png">

macintoshpie reviewed (approved):
😎  very nice
